packages/select/src/common/listItemsProps.ts

```diff
@@ -98,6 +98,13 @@ export interface IListItemsProps<T> extends IProps {
      */
     onQueryChange?: (query: string, event?: React.ChangeEvent<HTMLInputElement>) => void;
 
+    /**
+     * Whether the active selection should be reset when a query is entered.
+     * As you query, the first matching item will become the active selection.
+     * @default true
+     */
+    resetOnQuery?: boolean;
+
     /**
      * Whether the querying state should be reset to initial when an item is
      * selected (immediately before `onItemSelect` is invoked). The query will
```

Comment:
> Whether the active item should be reset to the first matching item when an item is selected.
The query will also be reset to the empty string.

and please make this same change in `resetOnClose`

Reply:
Not sure I follow -- you want that to replace the docs for `resetOnSelect` (in listItemProps.ts) & `resetOnClose` (in select.tsx)?

Reply:
that's the goal. i can do it if you're not comfortable